exercises/transpose/src/test/java/TransposeTest.java

```diff
@@ -0,0 +1,259 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.Before;
+
+import static org.junit.Assert.assertEquals;
+
+public class TransposeTest {
+    private Transpose transpose;
+
+    @Before
+    public void setup() {
+        transpose = new Transpose();
+    }
+
+    @Test
+    public void emptyString() {
+        String input = "";
+        String expected = "";
```

Comment:
You could maybe name them `inputMatrix` and `transposedMatrix`? But that's up to you, `input` and `expected is fine as well :)